notebooks/Analyze_Model_Outputs.ipynb

```diff
@@ -0,0 +1,3047 @@
+{
```

Comment:
<p>let -&gt; lets</p><p>"performed least" -&gt; "performed the least" - I think..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>This sentence was a mess. I've rewritten it as "We can use these per-document statistics to find documents where this model performed poorly."</p>